index.html

```diff
@@ -349,6 +341,43 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+	   <p>If the element, which is set as the target for mouse events while the pointer is
+	   locked, is a
+           <a href="https://dom.spec.whatwg.org/#concept-shadow-including-descendant">shadow-including descendant</a> of the
+           <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+	   return the result of the following algorithm:
+	   <dl>
```

Comment:
You can reference shadow DOM for this, sorry for confusing you!

If you did want to inline the algorithm, I think it's better to fit with the conventions of this spec, instead of shadow DOM's idiosyncratic CAPITAL LETTER VARIABLES and input/output `<dl>`.

But just referencing shadow DOM's algorith, like you did before, is fine. I just thought it's something that should move to DOM eventually, so I commented on that.